Q2.txt

```diff
@@ -0,0 +1,18 @@
+javascript:
+var a = ["john", "mike", "susan", "matt", "joe", "larry", "ken", "samantha", "jenny", "Sara", "henry"];
+var x=a.length -1;
+while (x>0){
+	x=x-2;
+	console.log(a[x].toUpperCase());
+	x=x-1;
```

Comment:
it seems redundant to do x - 2 then x - 1 in the same iteration. Just print a[x] then do x -= 3